src/basics.md

```diff
@@ -1415,10 +1442,14 @@ fn main() {
 [`File::open`]: https://doc.rust-lang.org/std/fs/struct.File.html#method.open
 [`File::try_clone`]: https://doc.rust-lang.org/std/fs/struct.File.html#method.try_clone
 [`File`]: https://doc.rust-lang.org/std/fs/struct.File.html
+[`FixedOffset`]:
+https://docs.rs/chrono/*/chrono/offset/struct.FixedOffset.html
 [`HashMap`]: https://doc.rust-lang.org/std/collections/struct.HashMap.html
 [`hmac::Signature`]: https://docs.rs/ring/*/ring/hmac/struct.Signature.html
 [`IndependentSample::ind_sample`]: https://doc.rust-lang.org/rand/rand/distributions/trait.IndependentSample.html#tymethod.ind_sample
 [`Lines`]: https://doc.rust-lang.org/std/io/struct.Lines.html
+[`Local::now`]:
```

Comment:
- please keep the reference links in a single line
- I'd suggest to qualify the name with at least one more hierarchy segment `chrono::offset::Local::now` or `offset::Local::now`